src/base/static/js/views/map-view.js

```diff
@@ -38,6 +39,12 @@
         self.initGeolocation();
       }
 
+      // TODO: only init if geometry editing is enabled?
```

Comment:
I don't think it's a big deal for now, but I'll leave this TODO in for a future iteration of the editor